.github/workflows/notify.yml

```diff
@@ -18,8 +18,8 @@ jobs:
           fetch-depth: 50
       - uses: erlef/setup-beam@v1
         with:
-          otp-version: 25.0
-          elixir-version: 1.14.0
+          otp-version: '26.0'
+          elixir-version: '1.14.4'
```

Comment:
I don't think we need to update this one because we don't build any publishing artifact.